corehq/apps/userreports/tasks.py

```diff
@@ -204,37 +205,31 @@ def _run_report(backend_to_use):
     return objects
 
 
-@periodic_task(
-    run_every=crontab(minute="*/5"),
-    queue=settings.CELERY_PERIODIC_QUEUE,
-)
-def queue_async_indicators():
-    start = datetime.utcnow()
-    cutoff = start + ASYNC_INDICATOR_QUEUE_TIME
-    time_for_crit_section = ASYNC_INDICATOR_QUEUE_TIME.seconds - 10
+@periodic_task(run_every=crontab(minute="*/5"), queue=settings.CELERY_PERIODIC_QUEUE)
+def run_queue_async_indicators_task():
+    queue_async_indicators.delay()
```

Comment:
I believe that max_retries on that task means that it will attempt to queue the task, fail because of a lock in serial_task and then no longer try again. @gcapalbo can you confirm that?